src/mock-server/handlers/cms/pages/fixtures/page/feedback.ts

```diff
@@ -160,6 +160,32 @@ export const feedbackMock: PageResponse<PageType.Feedback> = {
       choices: '',
       default_value: 'true',
     },
+    {
+      id: 11,
+      meta: {
+        type: 'forms.FormField',
+      },
+      clean_name: 'enter_a_memorable_date',
+      label: 'Enter a memorable date',
+      field_type: 'date',
+      help_text: 'For example, 31 3 1980',
+      required: true,
+      choices: '',
+      default_value: '',
+    },
+    {
+      id: 12,
+      meta: {
+        type: 'forms.FormField',
+      },
+      clean_name: 'enter_a_memorable_date',
+      label: 'Enter a memorable date',
+      field_type: 'datetime',
+      help_text: 'For example, 31 3 1980',
+      required: false,
+      choices: '',
+      default_value: '',
+    },
```

Comment:
I think you can delete the object 12 because we don't have a datetime field_type, we only have a date field type. 

If you set 'required' to true in the mock server then you are going to need to update the e2e tests so the the date is filled in as part of the tests in order to be able to submit the form. 

```suggestion
 {
      id: 11,
      meta: {
        type: 'forms.FormField',
      },
      clean_name: 'enter_a_memorable_date',
      label: 'Enter a memorable date',
      field_type: 'date',
      help_text: 'For example, 31 3 1980',
      required: false,
      choices: '',
      default_value: '',
    },
```